2022-09-05 11:07:58,076 INFO    MainThread:18496 [wandb_setup.py:_flush():71] setting env: {}
2022-09-05 11:07:58,077 WARNING MainThread:18496 [wandb_setup.py:_flush():71] could not find program at run.py
2022-09-05 11:07:58,077 INFO    MainThread:18496 [wandb_setup.py:_flush():71] setting login settings: {}
2022-09-05 11:07:58,077 INFO    MainThread:18496 [wandb_init.py:_log_setup():371] Logging user logs to /home/zzx/deepke/DeepKE/example/re/standard/logs/2022-09-05_11-07-57/wandb/run-20220905_110758-38o33anz/logs/debug.log
2022-09-05 11:07:58,077 INFO    MainThread:18496 [wandb_init.py:_log_setup():372] Logging internal logs to /home/zzx/deepke/DeepKE/example/re/standard/logs/2022-09-05_11-07-57/wandb/run-20220905_110758-38o33anz/logs/debug-internal.log
2022-09-05 11:07:58,077 INFO    MainThread:18496 [wandb_init.py:init():404] calling init triggers
2022-09-05 11:07:58,077 INFO    MainThread:18496 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-09-05 11:07:58,077 INFO    MainThread:18496 [wandb_init.py:init():449] starting backend
2022-09-05 11:07:58,077 INFO    MainThread:18496 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-09-05 11:07:58,089 INFO    MainThread:18496 [backend.py:ensure_launched():199] starting backend process...
2022-09-05 11:07:58,097 INFO    MainThread:18496 [backend.py:ensure_launched():204] started backend process with pid: 18562
2022-09-05 11:07:58,097 INFO    MainThread:18496 [wandb_init.py:init():458] backend started and connected
2022-09-05 11:07:58,099 INFO    MainThread:18496 [wandb_init.py:init():519] updated telemetry
2022-09-05 11:07:58,113 INFO    MainThread:18496 [wandb_init.py:init():549] communicating current version
2022-09-05 11:07:59,995 INFO    MainThread:18496 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.13.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-09-05 11:07:59,995 INFO    MainThread:18496 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-09-05 11:08:00,280 INFO    MainThread:18496 [wandb_init.py:init():592] starting run threads in backend
2022-09-05 11:08:01,885 INFO    MainThread:18496 [wandb_run.py:_console_start():1816] atexit reg
2022-09-05 11:08:01,886 INFO    MainThread:18496 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-09-05 11:08:01,886 INFO    MainThread:18496 [wandb_run.py:_redirect():1695] Redirecting console.
2022-09-05 11:08:01,888 INFO    MainThread:18496 [wandb_run.py:_redirect():1751] Redirects installed.
2022-09-05 11:08:01,888 INFO    MainThread:18496 [wandb_init.py:init():619] run started, returning control to user process
2022-09-05 11:08:02,058 INFO    MainThread:18496 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-09-05 11:08:02,058 INFO    MainThread:18496 [wandb_run.py:_restore():1758] restore
2022-09-05 11:08:04,899 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 669
}

2022-09-05 11:08:05,198 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 669
}

2022-09-05 11:08:05,300 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7814
}

2022-09-05 11:08:05,401 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7814
}

2022-09-05 11:08:05,502 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 669
  total_bytes: 7814
}

2022-09-05 11:08:05,604 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2938
  total_bytes: 7814
}

2022-09-05 11:08:05,705 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 4333
  total_bytes: 7814
}

2022-09-05 11:08:05,806 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7814
  total_bytes: 7814
}

2022-09-05 11:08:05,908 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7814
  total_bytes: 7814
}

2022-09-05 11:08:06,009 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7814
  total_bytes: 7814
}

2022-09-05 11:08:06,110 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7814
  total_bytes: 7814
}

2022-09-05 11:08:06,212 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7814
  total_bytes: 7814
}

2022-09-05 11:08:06,742 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7814
  total_bytes: 7814
}

2022-09-05 11:08:07,450 INFO    MainThread:18496 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7814
  total_bytes: 7814
}
local_info {
}

2022-09-05 11:08:08,568 INFO    MainThread:18496 [wandb_run.py:_append_files():2184] logging synced files
